src/App.jsx

```diff
@@ -2,53 +2,23 @@ import './App.scss';
 
 import todosFromServer from './api/todos.json';
 import usersFromServer from './api/users.json';
+import { TodoList } from './components/TodoList';
 
 function getUserById(userId) {
-  return usersFromServer.find(user => user.id === userId)
-      || null;
+  return usersFromServer.find(user => user.id === userId) || null;
 }
 
 export const todos = todosFromServer.map(todo => ({
-  ...todo,
-  user: getUserById(todo.userId),
+  ...todo, user: getUserById(todo.userId),
 }));
 
 export const App = () => (
   <div className="App">
     <h1 className="App__title">Static list of todos</h1>
-
     <section className="TodoList">
-      <article className="TodoInfo TodoInfo--completed">
-        <h2 className="TodoInfo__title">HTML</h2>
-
-        <a className="UserInfo" href="mailto:Sincere@april.biz">
-          Leanne Graham
-        </a>
-      </article>
-
-      <article className="TodoInfo TodoInfo--completed">
-        <h2 className="TodoInfo__title">CSS</h2>
-
-        <a className="UserInfo" href="mailto:Sincere@april.biz">
-          Leanne Graham
-        </a>
-      </article>
-
-      <article className="TodoInfo TodoInfo--completed">
-        <h2 className="TodoInfo__title">JS</h2>
-
-        <a className="UserInfo" href="mailto:Shanna@melissa.tv">
-          Ervin Howell
-        </a>
-      </article>
-
-      <article className="TodoInfo">
-        <h2 className="TodoInfo__title">React</h2>
-
-        <a className="UserInfo" href="mailto:Nathan@yesenia.net">
-          Clementine Bauch
-        </a>
-      </article>
+      {todos.map(todo => (
+        <TodoList todos={todo} key={todo.id} />
+      ))}
```

Comment:
```suggestion
      <TodoList todos={todos} />
```
You shouldn't create a new TodoList for each todo
![image](https://github.com/mate-academy/react_static-list-of-todos-js/assets/96409563/93ce3756-ae8d-4f50-8653-8e537a5a5b91)